string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,16 @@
+/* exported isVowel */
+/*
+create array of vowels capitalized. take arg and capitalize
+for every vowel in array, check if equal to arg. if true, then return true
+return false at the end when for loop finishes without returning true
+*/
+
+function isVowel(character) {
+  var vowels = ['A', 'E', 'I', 'O', 'U'];
+  for (var i in ['A', 'E', 'I', 'O', 'U']) {
```

Comment:
Why are you using a `for...in` loop on an array?

Reply:
even if order of array doesn't matter, should `for...in` be avoided for array looping?